source/puppet/4.9/style_guide.markdown

```diff
@@ -917,7 +917,7 @@ This places the values in the defaults hierarchy, which means that the defaults
 ```
 # using params.pp pattern < Puppet 4.9.0
 class my_module(
-  String $source = $mymodule::params::source,
+  String $source  = $mymodule::params::source,
   String $config  = $mymodule::params::config)  {
```

Comment:
This parameter is missing a trailing comma, and the `) {` should have a single space and be on a new line. Eg:
```puppet
  String $config  = $mymodule::params::config,
) {
```